src/components/domain/BeaconUseSection.tsx

```diff
@@ -149,7 +150,7 @@ const AboutThisUse: FunctionComponent<{ use: BeaconUse }> = ({
         value={use.cnOrMsnNumber}
       />
     )}
-    {use.dongle.includes("true") && (
```

Comment:
One of the changes in this PR is a move away from a blank `DraftRegistration` or `DraftUse` that has every field initialised with a blank string `""` to a domain object that has fields only if the user has added them (or `null` if the user had the opportunity but didn't).

`String.prototype.includes()` will therefore not work on values that the user hasn't entered (e.g. because they chose a Land use and so weren't prompted for a Purpose, or because they didn't complete an optional field).  In this case, if the user doesn't check the `dongle` checkbox then the value of `dongle` on `BeaconUse` is `null`.  Can't access string methods  on a null datatype.

This approach also gives us headroom to stop recording booleans as strings and record them as booleans instead (in a future PR).